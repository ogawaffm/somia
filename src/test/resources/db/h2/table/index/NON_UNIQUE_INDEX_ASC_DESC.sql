DROP TABLE IF EXISTS TEST.NON_UNIQUE_INDEX_ASC_DESC
/
CREATE TABLE TEST.NON_UNIQUE_INDEX_ASC_DESC
(
    COL_1 VARCHAR(10) NOT NULL,
    COL_2 VARCHAR(10) NOT NULL
)
/
CREATE INDEX TEST.NON_UNIQUE_IDX_ASC_DESC ON TEST.NON_UNIQUE_INDEX_ASC_DESC (COL_1 ASC, COL_2 DESC)
/