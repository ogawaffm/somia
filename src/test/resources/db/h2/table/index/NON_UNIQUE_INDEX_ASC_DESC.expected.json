{
  "name" : "NON_UNIQUE_INDEX_ASC_DESC",
  "catalogName" : "TEST",
  "schemaName" : "TEST",
  "typeName" : "BASE TABLE",
  "comment" : "",
  "columns" : [ {
    "name" : "COL_1",
    "type" : {
      "name" : "CHARACTER VARYING",
      "typeNumber" : 12,
      "className" : "java.lang.String",
      "precision" : 10,
      "scale" : 0,
      "isNullable" : false,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 0,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : false,
      "isCurrency" : false,
      "columnDisplaySize" : 10,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "COL_2",
    "type" : {
      "name" : "CHARACTER VARYING",
      "typeNumber" : 12,
      "className" : "java.lang.String",
      "precision" : 10,
      "scale" : 0,
      "isNullable" : false,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 0,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : false,
      "isCurrency" : false,
      "columnDisplaySize" : 10,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  } ],
  "indexes" : [ {
    "name" : "NON_UNIQUE_IDX_ASC_DESC",
    "columns" : [ {
      "name" : "COL_1",
      "order" : "ASC",
      "isIncluded" : false
    }, {
      "name" : "COL_2",
      "order" : "DESC",
      "isIncluded" : false
    } ],
    "isUnique" : false,
    "comment" : "",
    "qualifierName" : "TEST",
    "isClustered" : false,
    "isHashed" : false,
    "isStatistic" : false,
    "whereClause" : "",
    "cardinality" : 0,
    "pages" : 0
  } ],
  "collation" : "",
  "primaryKey" : null,
  "bestRowIdentifier" : null
}